app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailPresenter.java

```diff
@@ -46,12 +48,27 @@ public void attach(IEventDetailView eventDetailView, long initialEventId) {
         this.initialEventId = initialEventId;
     }
 
+    private void loadAll(boolean refresh) {
```

Comment:
Currently, we have this complex way to load both attendees and event which will change once you complete #148, so we won't even have refresh. Besides, the idea of start is to keep it consistent through all presenters, maybe I should make all presenters extend an interface with start and detach in them. Doing this so that if we create a general class to maintain lifecycle or use Android Architecture Components, we don't have to implement it in each class, so start, attach and detach should have same parameters throughout the framework